src/scss/config/_color.scss

```diff
@@ -140,9 +140,6 @@ $_ccs: (
 ///   See map for valid tokens.
 /// @return {*} -
 ///   Value from `$ccs` map, based on given key
-/// @example scss
```

Comment:
@mirisuzanne These are broken, I think until we find a way to pass a custom import resolver through Dart Sass via Herman.

Reply:
Makes sense. I wonder if we should just remove the annotation, and leave the examples in markdown prose?